force-app/main/default/lwc/errorSettings/errorSettings.js

```diff
@@ -69,6 +112,61 @@ export default class ErrorSettings extends LightningElement {
         this.template.querySelector("c-settings-save-canvas").handleHierarchySettingsChange(hierarchySettingsChange);
     }
 
+    handleSendErrorNotificationsChange(event) {
+        const eventDetail = event.detail;
+        this.showErrorNotificationRecipients = event.detail.value;
+
+        let hierarchySettingsChange = {
+            settingsType: "boolean",
+            settingsName: "Error_Notifications_On__c",
+            settingsValue: eventDetail.value,
+        };
+
+        this.template.querySelector("c-settings-save-canvas").handleHierarchySettingsChange(hierarchySettingsChange);
+    }
+
+    handleErrorNotificationRecipientCategoryChange(event) {
+        const eventDetail = event.detail;
+        this.errorNotificationRecipientCategory = eventDetail.value;
+        let hierarchySettingsChange;
```

Comment:
very minor too but this looks a bit inconsistent with the variable declaration in handleSendErrorNotificationsChange. I do like how we define it in handleSendErrorNotificationsChange.

Reply:
Altering to be more obvious.